packages/finschia-std/src/shim.rs

```diff
@@ -296,6 +326,8 @@ expand_as_any!(
     crate::types::cosmos::crypto::secp256r1::PubKey,
     crate::types::cosmos::crypto::ed25519::PubKey,
 );
+#[cfg(test)]
+expand_as_any!(crate::types::cosmwasm::wasm::v1::ClearAdminProposal,);
```

Comment:
```suggestion
expand_as_any!(
    // accounts have distincted structure
    crate::types::cosmos::auth::v1beta1::BaseAccount,
    crate::types::cosmos::auth::v1beta1::ModuleAccount,
    // pubkey required for base account
    // it can't be distinced by structure
    // so deserialing it back might not work properly
    crate::types::cosmos::crypto::secp256k1::PubKey,
    crate::types::cosmos::crypto::secp256r1::PubKey,
    crate::types::cosmos::crypto::ed25519::PubKey,
    // remove `#[cfg(test)]` when it is needed also in ordinarily (non-test) use
    #[cfg(test)]
    crate::types::cosmwasm::wasm::v1::ClearAdminProposal,
);
```